test/org/sagebionetworks/bridge/stormpath/StormpathAccountTest.java

```diff
@@ -58,28 +60,21 @@ public void setUp() throws Exception {
         data = new StubCustomData(); 
         when(account.getCustomData()).thenReturn(data);
 
-        String json = "{\"name\":\"Test\",\"birthdate\":\"1970-01-01\",\"imageData\":\"test\",\"imageMimeType\":\"image/png\"}";
-
         BridgeEncryptor encryptor1 = mock(BridgeEncryptor.class);
         when(encryptor1.getVersion()).thenReturn(1);
-        encryptDecryptValues(encryptor1, "111-222-3333", "111-222-3333-encryptor1encrypted");
-        encryptDecryptValues(encryptor1, "aHealthId", "aHealthId-encryptor1encrypted");
-        encryptDecryptValues(encryptor1, json, "json-encryptor1encrypted");
+        encryptDecryptValues(encryptor1, "1");
         
         BridgeEncryptor encryptor2 = mock(BridgeEncryptor.class);
         when(encryptor2.getVersion()).thenReturn(2);
-        encryptDecryptValues(encryptor2, "111-222-3333", "111-222-3333-encryptor2encrypted");
-        encryptDecryptValues(encryptor2, json, "json-encryptor2encrypted");
-        encryptDecryptValues(encryptor2, "555-555-5555", "555-555-5555-encryptor2encrypted");
+        encryptDecryptValues(encryptor2, "2");
         
         // This must be a passthrough because we're not going to set the signature through
         // StormpathAccount, we're going to put a legacy state in the map that's stubbing out
         // The CustomData element, and then verify that we can retrieve and deserialize the consent
         // even without a version attribute.
-        sig = new ConsentSignature.Builder().withName("Test").withBirthdate("1970-01-01").withImageData("test")
-                .withImageMimeType("image/png").withSignedOn(UNIX_TIMESTAMP).build();
+        sig = new ConsentSignature.Builder().withName("Test").withBirthdate("1970-01-01").withImageData("test").withImageMimeType("image/png").withSignedOn(UNIX_TIMESTAMP).build();
         legacySignature = MAPPER.writeValueAsString(sig);
-        encryptDecryptValues(encryptor2, legacySignature, legacySignature);
+        data.put("foo_consent_signature", legacySignature);
```

Comment:
Added test with only the consent list, consisting of a withdrawn consent and an active consent, that works.